src/ProjectTemplates/Web.ProjectTemplates/content/Grpc-Application/.template.config/template.json

```diff
@@ -5,10 +5,10 @@
     "Web",
     "gRPC"
   ],
-  "name": "ASP.NET Core gRPC Application",
+  "name": "ASP.NET Core gRPC App",
```

Comment:
? App?

Reply:
All other templates use the shorter form of App instead Application:
![image](https://user-images.githubusercontent.com/2030323/53067121-b09fc200-3487-11e9-8cd9-98f66fdcc0af.png)

Reply:
Gross

Reply:
Gross

Reply:
Gro